tools/system_libs.py

```diff
@@ -86,7 +86,7 @@ def create_lib(libname, inputs):
     building.emar('cr', libname, inputs)
 
 
-def get_wasm_libc_rt_files():
+def get_wasm_libc_rt_files(is_optz=False, all=False):
```

Comment:
I'm having trouble understanding the meaning of `all` here. Perhaps a comment would help? Even reading the code below I'm not sure why it is needed, it seems like `is_optz` should be a variation on the library basically.

Speaking of variations, we do have `OptimizedAggressivelyForSizeLibrary` which this could use? That would automatically provide the C define. That could then be used in the main .c file to do an ifdef on that and include the `_small` version etc. - would that be simpler?

Reply:
Yes, this is using the `OptimizedAggressivelyForSizeLibrary` method.  Indeed the *only* library we currently compile in the mode is `libc-rt` and its the `self.is_optz` on this library that signals if we are in this mode.

This is a freestanding function that the `libc-rt` library then calls and passes along the paramater.

Reply:
The reason this is a freestanding function is that it is used in two different places.   In libc-rt its used to select which math functions to compile, and int libc its used to select which math files to ignore.

Reply:
I added a comment which I hope will make this more clear.  I also renamed the `all` argument to `superset`.